purchase_order_line_description_adj/models/purchase.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models
+
+
+class Purchase(models.Model):
+    _inherit = "purchase.order.line"
+
+    def _get_product_purchase_description(self, product_lang):
+        display_default_code = self.env.company.po_display_default_code
+        if not display_default_code:
+            self.ensure_one()
+            name = product_lang.with_context(display_default_code=False).display_name
+            if product_lang.description_purchase:
+                name += "\n" + product_lang.description_purchase
+            return name
+        return super()._get_product_purchase_description(product_lang)
```

Comment:
```suggestion
        product_lang = product_lang.with_context(display_default_code=False)
        return super()._get_product_purchase_description(product_lang)
```
Can't we simply do this without the configuration?